sqlparser/token.go

```diff
@@ -777,6 +778,13 @@ func (tkn *Tokenizer) scanLiteralIdentifier() (int, []byte) {
 	if buffer.Len() == 0 {
 		return LEX_ERROR, buffer.Bytes()
 	}
+
+	// Double-quoted identifiers in PostgreSQL are case-sensitive.
+	// We need to remember which identifiers were wrapped with quotes so the query
+	// can be properly recreated later from the AST using .Format() method.
+	if quoteSeen != nil && tkn.IsPostgreSQL() {
```

Comment:
mysql supports quoted identifiers too - https://dev.mysql.com/doc/refman/8.0/en/identifiers.html#:~:text=Certain%20objects%20within%20MySQL%2C%20including,names%20are%20known%20as%20identifiers. With ANSI_QUOTES parameter. And we keep in mind that parameter in several places too.